ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -34,19 +34,22 @@
     onRender: function() {
         if (this.model.get('type') === 'basic' || this.model.get('type') === 'text') {
             this.$el.addClass('is-static');
-        }
-        else{
-        this.workspaceActions.show(new SearchFormInteractionsDropdownView({
-            model: new DropdownModel(),
-            modelForComponent: this.model,
-            dropdownCompanionBehaviors: {
-                navigation: {}
+        } else {
+            if (!this.isSystemTemplate) {
```

Comment:
Probably want to actually call this function `!this.isSystemTemplate()`

Reply:
Might as well refactor `isSystemTemplate` to not require a model argument as well since it has access to the `this` context and thus `this.model`.